apps/studio/src/features/editing-experience/components/PageCreateModal.tsx

```diff
@@ -67,11 +68,18 @@ export const PageCreateModal = ({
     setPageUrlLen(watchAllFields.url?.length || 0)
   }, [watchAllFields])
```

Comment:
for learning/out of scope of this PR, but same comment above. you do not need an effect, can just derive the variable. 

```tsx
const titleLen = watchedFields.title?.length ?? 0
const pageUrlLen = watchedFields.url?.length ?? 0
```